roles/rook/defaults/main.yml

```diff
@@ -0,0 +1,71 @@
+---
+##########################
+# operator
+
+operator_user: dragon
+operator_group: "{{ operator_user }}"
+
+##########################
+# rook
+
+container_registry_rook: quay.io
+rook_image: "{{ container_registry_rook }}/ceph/ceph"
+# renovate: datasource=docker depName=index.docker.io/rook/ceph
+rook_image_tag: "v17.2"
+
+## work directory inside the osism-ansible container:
+rook_work_directory: /tmp/rook/configuration
+## template directory containing rook CRD `*.yml` files, e.g. OSISM configuration directory
+rook_template_directory: "{{ role_path }}/templates"
+## kubernetes namespace in which the rook cluster should be installed
+## also see https://rook.io/docs/rook/v1.13/Storage-Configuration/Advanced/ceph-configuration/?h=#using-alternate-namespaces
+rook_namespace: rook-ceph
+rook_cluster_name: rook-ceph
+rook_mon_count: 3
+rook_mds_count: 2
+rook_mgr_count: 2
+rook_mgr_modules:
+  - name: balancer
+    enabled: true
+  - name: status
+    enabled: true
+  - name: prometheus
+    enabled: true
+rook_dashboard_enabled: true
+# https://rook.io/docs/rook/latest-release/Storage-Configuration/Monitoring/ceph-monitoring/
+rook_monitoring_enabled: false # TODO: enable 
+rook_metrics_disabled: false
+rook_network_encryption: true
+rook_network_compression: true
+rook_network_require_msgr2: false
+rook_network_public: ""
+rook_network_cluster: ""
```

Comment:
Use `rook_network_cluster: "{{ rook_network_public }}"` as default. This way we only have to add the `rook_network_public` to the configuration repository.

Reply:
:+1: